src/actions/surveySubmission.js

```diff
@@ -1,22 +1,29 @@
 import * as types from '.';
 
 
-export function retrieveSurveySubmissions(surveyId = null) {
+export function retrieveSurveySubmissions(surveyId = null, linked = null, page = 0, perPage = 20) {
     return ({ dispatch, getState, z }) => {
-        let orgId = getState().org.activeId;
+        const filters = [];
+        const orgId = getState().org.activeId;
         let promise;
 
+        if (linked !== null) {
+            linked = linked? ((filters.linked == 'linked')? 1 : 0) : null;
```

Comment:
Style note: the positioning of the `?` operator in the ternary conditional tripped me up. "Is this some Babel precompile feature?"

Reply:
Yeah, multi-level ternaries are not a great style choice. I'll reconsider in the future. :)